frontend/lib/norent/homepage.tsx

```diff
@@ -7,13 +7,44 @@ import { OutboundLink } from "../analytics/google-analytics";
 import { Link } from "react-router-dom";
 import { NorentLogo } from "./components/logo";
 import { NorentFaqsPreview } from "./faqs";
+import classnames from "classnames";
 
 type NorentImageType = "png" | "svg";
 
 export function getImageSrc(name: string, type?: NorentImageType) {
   return `frontend/img/norent/${name}.${type || "svg"}`;
 }
 
+export const JumpArrow = (props: { to: string }) => (
+  <Link to={props.to} className="jf-jump-arrow">
+    <StaticImage
+      ratio="is-32x32"
+      src={getImageSrc("downarrow")}
+      alt="Explore the tool"
```

Comment:
Ooo nice alt text!! 👍  The only thing I'd suggest is that maybe have the alt text be a required prop instead, since it seems like its meaning might be contextual based on what it's linking to?  e.g. it would be unfortunate if this were to be reused in, say, an FAQ page, where the meaning might really be "explore the FAQs" or somesuch, but it still reads as "Explore the tool" to visually impaired users.

Reply:
👍